Mathlib/Data/Nat/Factors.lean

```diff
@@ -300,6 +300,23 @@ theorem eq_two_pow_or_exists_odd_prime_and_dvd (n : ℕ) :
           hprime.eq_two_or_odd'.resolve_right fun hodd => H ⟨_, hprime, hdvd, hodd⟩⟩
 #align nat.eq_two_pow_or_exists_odd_prime_and_dvd Nat.eq_two_pow_or_exists_odd_prime_and_dvd
 
+theorem four_dvd_or_exists_odd_prime_and_dvd_of_two_lt {n : ℕ} (n2 : 2 < n) :
+    4 ∣ n ∨ ∃ p, p ∣ n ∧ Prime p ∧ Odd p := by
```

Comment:
It would be nice to either make the ordering of this lemma match the previous, either by changing the previous lemma or by changing this lemma.